lib/app/features/feed/views/pages/feed_page/components/stories/components/story_list_item.dart

```diff
@@ -36,63 +31,71 @@ class StoryListItem extends HookWidget {
 
   static double get borderSize => 2.0.s;
 
+  static const _defaultAvatarUrl = 'https://i.pravatar.cc/150?u=@me';
```

Comment:
I think `Avatar` widget support to show a `DefaultAvatar` when it has no image. could we use it?

Reply:
updated